Code-the-Dream-School/bb-practicum-team-3-front
# add User Profile page

## Description

I added a profile page, where a user can switch between two tabs: personal details and reservations. "Personal details" are the primary information about the account holder ( first name, last name and email address). "Reservations" section displays a booking overview. "See details" link brings a user to the next page and provides all the booking details. Also a user can cancel their reservation by clicking "Cancel reservation" button or get back to the previous page by clicking "Return to reservation overview" button.

## Related Issue



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After

![image](https://user-images.githubusercontent.com/28917654/236658715-8c476e1f-41e9-4b3e-8123-396547ab0c64.png)

![image](https://user-images.githubusercontent.com/28917654/236658784-ae99e31a-208f-4f40-b915-9bbf8589cdb6.png)

![image](https://user-images.githubusercontent.com/28917654/236658823-3c7ef056-aee8-4514-b63a-ed4d58fb5ec1.png)


## Testing Steps / QA Criteria

sign up for the account (if you are a new user) or get logged in (if you already have an account)
click on "Profile" button in the Header section
click on "Personal details" tab, you should see user's first name, last name, email
click on "Reservations" tab , you should see either "It appears that you haven't made any reservations yet" or reservations details (if you did make a booking), for more reservation details click  the "See details" link. 
click on "Cancel reservation", check "Reservation" section after that, your reservation is supposed to be removed
